### ISSUE TITLE
Buttons not accessible on Android

### ISSUE THREAD

**Issue by tdekoning** [software-mansion/react-native-gesture-handler#1251]
## Description

The button components (like `BorderlessButton` and `RectButton`) are not clickable using the Talkback feature on Android.

## Steps To Reproduce

1. Use code listed below (this is generating using `react-native init` with added `react-native-gesture-handler` as a dependency)
2. Enable Talkback on your Android phone
3. Attempt to click on the buttons using the talkback feature

### Expected behavior
I expect the buttons "Rect button" and "Borderless button" to trigger their `onPress` callbacks, just like the "My button" does (which is from react-native itself)

### Actual behavior
No `onPress` is being triggered on the react-native-gesture-handler buttons.

## Snack or minimal code example

```
/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow strict-local
 */

import React from 'react';
import {
  SafeAreaView,
  ScrollView,
  View,
  Text,
  StatusBar,
  Button
} from 'react-native';

import {RectButton, BorderlessButton} from 'react-native-gesture-handler'

const App: () => React$Node = () => {
  return (
    <>
      <StatusBar barStyle="dark-content" />
      <SafeAreaView>
        <ScrollView
          contentInsetAdjustmentBehavior="automatic">
            {/*Does not work*/}
            <RectButton onPress={() => console.log('press rect button')}>
              <View accessible>
                <Text>Rect button</Text>
              </View>
            </RectButton>
            {/*Does not work*/}
            <BorderlessButton onPress={() => console.log('press borderless button')}>
              <View accessible>
                <Text>Borderless button</Text>
              </View>
            </BorderlessButton>
            {/*Works*/}
            <Button title="My button" onPress={() => console.log('react-native button')}/>
        </ScrollView>
      </SafeAreaView>
    </>
  );
};

export default App;
```

## Package versions

- React: 16.13.1
- React Native: 0.63.4
- React Native Gesture Handler: ^1.9.0

**Comment by Guuz** [comment]
Just wanna add the docs for this, as they describe exactly that this ***should*** work:
https://docs.swmansion.com/react-native-gesture-handler/docs/component-buttons
![image](https://user-images.githubusercontent.com/810327/100753519-2d482a00-33ea-11eb-9794-9dc24c2cb8f6.png)

**Comment by jakub-gonet** [comment]
This is documented, as linked above. 

We're considering applying accessible prop automatically to be compatible with RN's behavior.

**Comment by tdekoning** [comment]
@jakub-gonet it is documented, yet it doesn't work. How come this issue is set to closed?

Like my code snippet stated, i've added the accessibility view just as the documentation states, but the buttons do not trigger the `onPress` method when pressed with Talkback.

**Comment by tdekoning** [comment]
@jakub-gonet Can you please reopen this issue as it's been closed while this is an actual bug

**Comment by jakub-gonet** [comment]
Sure, I assume you can't do it yourself if a member closes your issue? Not sure about Github's behavior on this interaction.

**Comment by luizwhite** [comment]
Since the RectButtonProperties was deprecated and RectButtonProps took place, I have this error
So downgrading to react-native-gesture-handler@1.8.0 solved my problem @tdekoning

**Comment by jakub-gonet** [comment]
> Since the RectButtonProperties was deprecated and RectButtonProps took place, I have this error

That's our bad, we should've aliased old type to the new. Will fix it shortly, thanks for the report.

**Comment by tdekoning** [comment]
@jakub-gonet This issue has gone off of my own radar, but i'm still encountering issues with accessibility and buttons in react-native-gesture-handler. 

Here is an Expo snack that demonstrates the issue. The "My Button" is accessible, because it is a react-native component, while the others (react-native-gesture-handler) are not clickable.

**Comment by dhleong** [comment]
Running into this with the AccessibilityInspector on the iOS Simulator as well. The view is accessible and has the correct labels, but the press action does nothing.

If we wanted to look into this, any tips on where to start?

**Comment by j-piasecki** [comment]
Hi! I tried the snack by @tdekoning and while it's not working on Expo 42 (as selected in the snack), it does for me after changing the Expo SDK version to 44 (keep in mind you need to add `<GestureHandlerRootView>` to the top of the hierarchy then). Could you check if the issue still persists after upgrading?

**Comment by taylorkline** [comment]
I believe this is fixed since #1599. I am not seeing any issues locally.

**Comment by j-piasecki** [comment]
I think you're right, I'll close the issue then.
